ddtrace/profiling/recorder.py

```diff
@@ -1,11 +1,11 @@
 # -*- encoding: utf-8 -*-
 import collections
+import threading
```

Comment:
`as ddtrace_threading` ?

Reply:
I don't think it is necessary here. The only place where it would be required is where we have the need to look up the actual `threading` module used by the application from `sys.modules`, which we would assign to the variable `threading`.

Reply:
I am just wondering for consistency, being clear about which/when we have a cloned version of `threading` vs the non-cloned one.

Reply:
Ah I see. I guess you're suggesting doing this for every module that we need to disambiguate? Otherwise we would have to do it for every module that we import.

Reply:
yeah, I want to make sure the code is clear when we review/update the code later. are we dealing with one of our clones or the shared version?

Reply:
I think the issue is that we have other places that do import threading as ddtrace_threading because we also need to import the non-cloned version to iterate/check for all active threads.

So my comment here was mostly about, should we have some kind of consistent/not-just-"threading" name to indicate in the code when we are using (at least expected to be using) a cloned module.

Do we think we could end up getting confused in the future by what is cloned vs not when reviewing/debugging code?


(not a strong opinion here, just musings)

Reply:
I agree with @Yun-Kim that it's sufficient for contributors to understand that modules imported in `ddtrace/` are probably not accessible to user code. I've committed a suggestion that splits the difference with an explanatory comment for contributors. https://github.com/DataDog/dd-trace-py/pull/5105/commits/20929f246dd5973d60f02e59049abdc185857c70

Reply:
👍🏻 would be good to cover in a central architecture doc if/when we get one too